example/db-migration-tf/main.tf

```diff
@@ -31,7 +31,3 @@ module "db_migration" {
   vpc_id                      = var.vpc_id
   rds_security_group_id       = local.rds_security_group
 }
-
-output "db_migration" {
```

Comment:
Was this just not necessary?

Reply:
Yeah, nothing was being returned so it just said `null`